.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: KpgnfDK8BDSSTSS4lSrCvHwHnchLgIQ0D
```

Comment:
Will this link up with a GitHub Hook? I don't quite understand how to get coveralls working, definitely a good idea though.

Reply:
This was an attempt to get it working. Still fairly confused but will try and get it to work. It's an action on materialsproject/pymatgen, and I don't understand why that isn't run here

Reply:
Ahh, that would have to be set up on my repo. That's my responsibility so for now I'd just remove this.

Later on, (read: post-qual) we'll break out all this code into a separate repo. At that point, we'll set up CI + coveralls.